lib/assesments/assessment_2/fi_a2_muhammad_bilal/fi_a2_mbilal_profile/fi_a2_mbilal_profile.dart

```diff
@@ -0,0 +1,174 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+
+class BilalProfileScreen extends StatefulWidget {
+  const BilalProfileScreen({Key? key}) : super(key: key);
+
+  @override
+  State<BilalProfileScreen> createState() => _BilalProfileScreenState();
+}
+
+class _BilalProfileScreenState extends State<BilalProfileScreen> {
+  List<String> text1 =['Photos','Followers','Follows'];
+  List<String> text2 =['315','77.5k','500'];
+  List<String> text3=['Photos','Video','Tagged'];
+  List<String> picture=['assets/fi_a2_mbilal_assets/pic2.png','assets/fi_a2_mbilal_assets/pic3.png','assets/fi_a2_mbilal_assets/pic4.png'];
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+     body:Column(
+       children: [
+         Padding(
+           padding: const EdgeInsets.only(top: 60,left: 30,right: 36),
+           child: Row(
+             mainAxisAlignment: MainAxisAlignment.spaceBetween,
+             children: [
+               SvgPicture.asset('assets/fi_a2_mbilal_assets/Go back.svg',
+                 width: 27,height:19,),
+               const Text('My Profile',
+                 style: TextStyle(
+                   fontSize: 23,
+                   fontWeight: FontWeight.w700,
+                   fontFamily: 'Inter',
+                 ),),
+               SvgPicture.asset('assets/fi_a2_mbilal_assets/Settings.svg',
+                 width: 27,height: 28,),
+             ],
+           ),
+         ),
+         const Padding(
+           padding: EdgeInsets.only(top:40),
+           child: Center(
+             child: CircleAvatar(
+               backgroundImage:AssetImage('assets/girl1.png'),
+               radius: 50,
+             ),
+           ),
+         ),
+         const Padding(
+           padding: EdgeInsets.only(top:24),
```

Comment:
If this padding is for space pleas use a sizedBox

Reply:
ok